cmake/FooyinMacros.cmake

```diff
@@ -1,11 +1,38 @@
 function(fooyin_set_rpath target prefix)
     get_filename_component(path "${CMAKE_INSTALL_PREFIX}/${prefix}" ABSOLUTE)
     file(RELATIVE_PATH relative ${path} "${CMAKE_INSTALL_PREFIX}/${LIB_INSTALL_DIR}")
-    set(rpath "\$ORIGIN/${relative}")
```

Comment:
Not sure about this. `${relative}` was empty when building in CI, which made fooyin binary unable to find dylibs